fetch.js

```diff
@@ -311,7 +311,13 @@
       xhr.onload = function() {
         var status = (xhr.status === 1223) ? 204 : xhr.status
         if (status < 100 || status > 599) {
```

Comment:
This check is in place only for IE 9, which can set `xhr.status` to invalid values. Unless another browser is reaching this block unintentionally, I think we can close this pull request.